pkg/skaffold/schema/latest/config.go

```diff
@@ -279,6 +282,22 @@ type DockerConfig struct {
 	SecretName string `yaml:"secretName,omitempty"`
 }
 
+// ResourceRequirements describes the resource requirements for the kaniko pod
+type ResourceRequirements struct {
+	// Requests https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#resource-requests-and-limits-of-pod-and-container
+	Requests *ResourceRequirement `yaml:"requests,omitempty"`
+	// Limits https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#resource-requests-and-limits-of-pod-and-container
+	Limits *ResourceRequirement `yaml:"limits,omitempty"`
+}
+
+// ResourceRequirement see https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#resource-requests-and-limits-of-pod-and-container
+type ResourceRequirement struct {
+	CPU    string `yaml:"cpu,omitempty"`
```

Comment:
Can you add comments for CPU and Memory?

Reply:
done